templates/jitsi_meet_config.js.j2

```diff
@@ -350,10 +350,10 @@ var config = {
         // through the JVB and use the peer to peer connection instead. When a
         // 3rd participant joins the conference will be moved back to the JVB
         // connection.
-        enabled: true,
+        enabled: {{ jitsi_meet_enable_p2p_mode | to_json }},
 
         // Use XEP-0215 to fetch STUN and TURN servers.
-        useStunTurn: {{ jitsi_meet_use_stun_turn }},
+        useStunTurn: {{ jitsi_meet_use_stun_turn | to_json}},
```

Comment:
Here it would be useful add a space before closing the mustaches to make the Ansible lint happy

Reply:
use | bool  | lower instead of to_json